public/collections.json

```
@@ -1,4 +1,28 @@
 {
+  "Generative AI": {
+    "id": "genAI",
+    "languages": ["en"],
+    "jurisdictions": ["EU"],
```

Just remembered we have 2 Chinese services!
```suggestion
    "jurisdictions": ["EU", "CN"],
```